ScribiteModuleInstaller.php

```diff
@@ -9,6 +9,7 @@ class ScribiteModuleInstaller extends AbstractExtensionInstaller
     public function install()
     {
         $this->setVar('DefaultEditor', 'CKEditor');
+        $this->setVar('UploadDirectory', 'userdata/uploads');
```

Comment:
This is `userdata/uploads` in 1.5.x but `web/uploads` in 2.x.
The core defines the `datadir` parameter for this https://github.com/zikula/core/blob/2.0/src/app/config/parameters.yml#L9 which should be used rather than a hardcoded value.

Reply:
Ok, need to examine this.

Reply:
To be able to preparte this PR, I installed and test in Zikula 1.5.
In this regard, what is the correct version of Scribite that should be used with Zikula 1.5?

Reply:
`$this->setVar('UploadDirectory', $this->container->getParameter('datadir'));`